src/TableContent.js

```diff
@@ -2,8 +2,17 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { Table } from 'evergreen-ui'
 import URI from 'urijs'
+import ReactTooltip from 'react-tooltip'
 
 import tableHeaderCells from './tableHeaderCells'
+import TooltipIcon from './TooltipIcon'
+
+const downloadTooltip = `<ul>
```

Comment:
I can't remember the discussion in the planning meeting, but did we agree we'll pass these in as props? This tooltip won't be exactly the same for bulk Atlas.

Reply:
I do not think bulk Atlas is going to use download function yet. The question mark is a temporal solution as we discussed, but I agree a props is always good.